### PR TITLE
Using the Flask-MongoEngine model_form to create nice forms.

### PR DESCRIPTION
**Pull request by jzarob**

### CODE REVIEW

File: angel_tree/mod_family/models.py [hunk 1]
```diff
@@ -1,52 +1,45 @@
...
-    name = MEStringField(min_length=1)
-    # TODO: Modify based on requirements.
-    age = IntField(min_value=3, max_value=13)
-    gender = MEStringField(required=True)
-    pant_size = MEStringField(required=True)
-    shirt_size = MEStringField(required=True)
-    wish = MultiLineStringField(required=True)
-    other = MultiLineStringField(required=False)
+    name = MEStringField(min_length=1, max_length=500, verbose_name="Name")
+    age = IntField(min_value=3, max_value=13, verbose_name="Age")
```
Comment (aeewhite) [inline angel_tree/mod_family/models.py]: So do these values get validated on form submit?

File: angel_tree/mod_settings/models.py [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-class SettingsForm(FlaskForm):
-    test_setting = StringField('Test Setting', [validators.required(), validators.length(max=50)])
-    submit = SubmitField("Save Settings")
+class Settings(Base):
+    test_setting = MEStringField(required=True, max_length=50, verbose_name="Test Setting")
 
 
-class Settings(Base):
-    test_setting = MEStringField(required=True)
+class SettingsForm(model_form(Settings, exclude=['date_modified', 'date_added'])):
```
Comment (aeewhite) [inline angel_tree/mod_settings/models.py]: Could probably do with a comment here explaining what this magic line does

File: requirements.txt [hunk 3]
```diff
@@ -1,12 +1,14 @@
 appdirs==1.4.0
 bcrypt==3.1.2
+blinker==1.4
```
Comment (aeewhite) [inline requirements.txt]: Did we determine what this is used for?

File: requirements.txt [hunk 4]
```diff
@@ -20,6 +22,7 @@ mock==2.0.0
 mongoengine==0.11.0
 mongomock==3.7.0
 nose==1.3.7
+numpy==1.12.0
```
Comment (aeewhite) [inline requirements.txt]: I think Matt will eventually need this too

File: angel_tree/templates/family/index.html [hunk 5]
```diff
@@ -36,6 +36,9 @@
                     <td>
                         <a href="{{ url_for("family.family_form", family_id=family.id) }}">
                             <span class="glyphicon glyphicon-pencil"></span> Edit
+                        </a> |
+                        <a href="{{ url_for("family.delete_family", family_id=family.id) }}">
+                            <span class="glyphicon glyphicon-trash"></span> Delete
```
Comment (aeewhite) [inline angel_tree/templates/family/index.html]: We need to put some sort of confirmation on this before we hand it off to the Salvation Army folks

**Comment by aeewhite** [comment]
I'm really surprised the code coverage went down with these changes

**Comment by aeewhite** [comment]
I'm surprised the code coverage went down with these changes

File: angel_tree/mod_family/models.py (hunk 1 above)
Comment (jzarob) [inline angel_tree/mod_family/models.py]: yes

**Pull request merged**